packages/fluence-connection/src/Buffer.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { isBrowser } from 'browser-or-node';
+import { Buffer as BufferPolyfill } from 'buffer';
+
+export default isBrowser ? BufferPolyfill : Buffer;
```

Comment:
Are you completely sure all of this in needed? As I remember, this polyfill should just work if you import it without any additional stuff. Please make sure we are not doing extra work here